members/people.json

```diff
@@ -72725,8 +72725,6 @@
       "start_date": "2007-10-18"
     },
     {
-      "end_date": "2021-01-04",
```

Comment:
He is according to https://members.parliament.uk/member/630/career? 
(Lord Wallace of Tankerness, this is)

Reply:
Oh he switched then switched *back*. That makes sense of that.

Reply:
Will adjust.

Reply:
New commit to add the switch back to lib dems.